### DISCUSSION TITLE
Adding Git repo details to material mkdocs

### DISCUSSION THREAD

**Discussion by kajulnisha9721** [squidfunk/mkdocs-material#5795] [Q&A] [answered]
how to add git repo details to show the fork details under repo url
![image](https://github.com/squidfunk/mkdocs-material/assets/83535981/f4633914-9e71-4cb8-a98b-fb4411694036)
This is my mkdocs.yml file,
repo_name: squidfunk/mkdocs-material
repo_url: https://github.com/squidfunk/mkdocs-material
theme:
  icon:
    repo: fontawesome/brands/git-alt

**Comment by HonkingGoose** [comment, marked as answer]
Read the [Material for MkDocs, adding a Git repository](https://squidfunk.github.io/mkdocs-material/setup/adding-a-git-repository/) page first. Then try this in your `mkdocs.yml` file:

```yaml
repo_name: jimandreas/mkdocs-material
repo_url: https://github.com/jimandreas/mkdocs-material
```
